### PR TITLE
More G-Mode Setups in Unheated Norfair

### PR DESCRIPTION
**Pull request by kjbranch**
The only one missing now is Bubble Mountain.

### CODE REVIEW

File: region/norfair/crocomire/Post Crocomire Jump Room.json [hunk 1]
```diff
@@ -738,6 +738,36 @@
         "Then you need to manipulate it to go high enough to be used as a stepping stone once frozen."
       ]
     },
+    {
+      "link": [5, 1],
+      "name": "G-Mode Setup - Get Hit By Mella (Long Manipulaton)",
```
Comment (kjbranch) [inline region/norfair/crocomire/Post Crocomire Jump Room.json:743]: I wasn't able to test this one, so I just copied the strat that went to the item. Can someone else test it?

File: region/norfair/crocomire/Post Crocomire Shaft.json [hunk 2]
```diff
@@ -360,9 +398,15 @@
       "name": "Viola Ice Clip Door Lock Skip",
       "requires": [
         {"ammo": {"type": "Super", "count": 2}},
-        "canTrickyUseFrozenEnemies",
```
Comment (osse101) [inline region/norfair/crocomire/Post Crocomire Shaft.json:363]: This still requires Ice

File: region/norfair/east/Red Pirate Shaft.json [hunk 3]
```diff
@@ -174,6 +189,17 @@
       "requires": [
         "h_canCrystalFlash"
       ]
+    },
+    {
+      "link": [2, 2],
+      "name": "G-Mode Setup - Get Hit By Geemer",
+      "requires": [
+        {"ammo": {"type": "Super", "count": 1}}
```
Comment (osse101) [inline region/norfair/east/Red Pirate Shaft.json:197]: Dodging pirates is probably a safe assumption

**Comment by kjbranch** [comment]
I believe that addresses the feedback. I'm mostly unsure if I got the Mella strat correct.

**Pull request merged**